### DISCUSSION TITLE
Lazy loading doesn't work for the first search

### DISCUSSION THREAD

**Discussion by synic** [folke/flash.nvim#234] [Q&A] [answered]
Maybe this plugin just can't be fully lazy loaded?

If you leave it with the default configuration for lazy.vim, and you type `/` to start a search, you won't get the jump options until your second search, presumably because the plugin is loaded at that point.

Is there any way to work around this?

**Comment by asyncedd** [comment, marked as answer]
try putting `event = "CmdlineEnter"` in your plugin spec
